USFMToolsSharp/Models/Markers/MSMarker.cs

```diff
@@ -9,12 +9,16 @@ namespace USFMToolsSharp.Models.Markers
     /// </summary>
     public class MSMarker : Marker

```

Comment:
This marker actually has contents. Just like here https://ubsicap.github.io/usfm/titles_headings/index.html#ms so either allow it to have a TextMarker or implement its parse.

Reply:
The content is being stored in it's 'Heading' property. What other content should it need to store?

Reply:
Totally missed that